build_tools/buildkite/scripts/download_artifacts_from_pipeline.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
What advantages does this script have over using the `buildkite-agent` command directly? https://buildkite.com/docs/pipelines/artifacts#downloading-artifacts

Reply:
Good idea! I misread [this section](https://buildkite.com/docs/pipelines/artifacts#downloading-artifacts-outside-of-a-running-build) and thought it was not possible.

Tested `buildkite-agent download` with build ID and it worked.

Reply:
Removed the custom download script.